src/Microsoft.Dnx.Tooling/Publish/PublishProject.cs

```diff
@@ -437,7 +448,7 @@ private static string CombinePath(string path1, string path2)
             return Path.Combine(path1, path2);
         }
 
-        private async Task<bool> Restore(PublishRoot root, PublishProject publishProject, string restoreDirectory)
+        private async Task<bool> Restore(PublishRoot root, PublishProject publishProject, string restoreDirectory, IEnumerable<FrameworkName> frameworks)
```

Comment:
Suggestion: I think the problem here is we have separate restore logic in publish which is different from `dnu restore`. Should we consider consolidate them together.

This is not supposed to block this PR.